### PR TITLE
<meta> tag head wash and DRY (fixes #230, etc.)

### LINKED ISSUE #230: Add `link[rel="canonical"]`
```html
<link rel="canonical" href="https://aframe.io/">
```

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: scripts/helpers.js [hunk 1]
```diff
@@ -117,6 +117,14 @@ hexo.extend.helper.register('website_github_edit_url', function (path) {
   }
 });
 
+hexo.extend.helper.register('external_url_for', function (path, options) {
```
Comment (ngokevin) [inline scripts/helpers.js]: docstring/comment

**Comment by ngokevin** [comment]
r+wc

File: _config.yml [hunk 2]
```diff
@@ -11,6 +11,7 @@ email: aframe@mozilla.com
 language:
 robots: all
 blog_title: Blog
+theme_color: black
```
Comment (ngokevin) [inline _config.yml]: What's this do

Comment (cvan) [inline _config.yml]: ```<meta name="theme-color" content="black">```

https://html.spec.whatwg.org/multipage/semantics.html#meta-theme-color
https://developers.google.com/web/fundamentals/design-and-ui/browser-customization/theme-color

Comment (ngokevin) [inline _config.yml]: k

File: scripts/helpers.js [hunk 3]
```diff
@@ -117,6 +117,17 @@ hexo.extend.helper.register('website_github_edit_url', function (path) {
   }
 });
 
+/**
```
Comment (cvan) [inline scripts/helpers.js]: updated

File: package.json [hunk 4]
```diff
@@ -21,7 +21,8 @@
     "hexo-server": "^0.2.0",
     "js-yaml": "^3.5.5",
     "moment": "^2.12.0",
-    "request": "^2.69.0"
+    "request": "^2.69.0",
+    "urljoin.js": "^0.1.0"
```
Comment (cvan) [inline package.json]: updated

**Pull request merged**